basyx.submodelrepository/basyx.submodelrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/submodelrepository/http/DummySubmodelRepositoryComponent.java

```diff
@@ -26,17 +26,13 @@
 
 package org.eclipse.digitaltwin.basyx.submodelrepository.http;
 
-import org.eclipse.digitaltwin.basyx.submodelrepository.InMemorySubmodelRepository;
-import org.eclipse.digitaltwin.basyx.submodelservice.InMemorySubmodelServiceFactory;
-import org.eclipse.digitaltwin.basyx.submodelservice.SubmodelServiceFactory;
+import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.context.annotation.Bean;
 
```

Comment:
Please add Javadoc similar to other Dummy components.

Reply:
done